.github/workflows/check-try-state.yml

```diff
@@ -24,17 +24,47 @@ jobs:
     steps:
       - name: Checkout sources
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
I believe Ermal pointed this part as redundant, right?

Reply:
No, I mean the other git checkout is redundant. I will add more comments

Reply:
https://github.com/AstarNetwork/Astar/pull/1499#discussion_r2198079446